bin/match_enc

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env ruby
```

Comment:
Personally I'm not a huge fan of abbreviations in the names of tools, maybe let's call it `match_encryption`? `match_file` could also work, it would read as `match_file encrypt`/`match_file decrypt`, which would be fairly intuitive.

Reply:
The name was originally to be close to `openssl enc`.  Your comment makes sense though.